intranet/apps/preferences/views.py

```diff
@@ -337,9 +337,11 @@ def preferences_view(request):
             preferred_pic_form = PreferredPictureForm(user, initial=preferred_pic)
             bus_route_form = BusRouteForm(initial=bus_route)
 
+            """
             privacy_options = get_privacy_options(user)
-            logger.debug(privacy_options)
             privacy_options_form = PrivacyOptionsForm(user, initial=privacy_options)
+            """
+            privacy_options_form = None
```

Comment:
I would disable the call to `save_privacy_options()` above, if only to be consistent. We might also want to add comments explaining why this is disabled.